projects/project.cpp

```diff
@@ -508,7 +494,7 @@ namespace projects {
       const CellID cellID = (int) ((x - Parameters::xmin) / dx) +
```

Comment:
Who coded that? (Me, likely??) Why don't we just read the ID from CellParams::??

Reply:
Yep, you committed that in 2014, I believe: https://github.com/ursg/vlasiator/commit/88cd0b4b7e413953d495d7d6003c9a4529720447#diff-ac171b423794277eb8b6b862f7037709aab19b0bdc5a38b974627241a30cb159R212

Reply:
So, are we changing this or not bothering?